examples/c_api/fragment_info.c

```diff
@@ -141,9 +141,13 @@ void get_fragment_info() {
   printf("The number of written fragments is %d.\n", num);
 
   // Get fragment name
-  const char* name;
-  tiledb_fragment_info_get_fragment_name(ctx, fragment_info, 0, &name);
-  printf("The fragment name is %s.\n", name);
+  tiledb_string_t* name;
+  const char* name_ptr;
+  size_t name_size;
+  tiledb_fragment_info_get_fragment_name_v2(ctx, fragment_info, 0, &name);
```

Comment:
It is poor programming practice not to check return values. Our examples should illustrate good programming practice.
The future-proof practice is to use `tiledb_status()` to extract the (legacy) status return value from the full C API return type. See the documentation of that function for why.

Reply:
Well, at the moment all C API examples don't check the return value… 🥲

Reply:
Well, let's start here. Created SC-25160 to track doing it for the rest of the examples.

Reply:
Done. What do you think?